### ISSUE TITLE
Unable to download files from Mac webview, Windows works fine. Is this a Webkit issue or CHOC?

### ISSUE THREAD

**Issue by Chesewip** [Tracktion/choc#39]
I basically put in a webview inside a JUCE plugin for accessing certain parts of my website, one being downloading the newest version of the plugin. (There's plenty of other ways I can do this so this is not a breaking issue). But as the title says, on windows the download works as expected, where as on mac, nothing happens. 

I wasn't sure if this is a webkit issue or something CHOC handles. Looking online I see some similar issues and I think it has something to do with a webkit callback. 

https://stackoverflow.com/questions/73257062/how-to-download-files-from-inside-wkwebview-on-macos

**Comment by julianstorer** [comment]
Yes, there do seem to be some places where WebKit needs code to manually handle things like this - I recently added something like that to handle file choosers. I'm too busy to look into this at the moment, but if anyone wanted to have a go, I could help review etc..
